test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -878,6 +1063,20 @@ private enum EmptyEnum
         {
         }
 
+        private enum EnumWithLocalizedDisplayNames
```

Comment:
Why does this `enum` use `Description` or `Prompt`? Only `Name` and `Group` area relevant.

Speaking of which: Think we need to do the same `IStringLocalizer` work for `Group`. Separate issue but worth filing.